Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM amazoncorretto:17
+COPY workflow-bot-app/build/libs/*.jar wdk.jar
+ENTRYPOINT ["java", "-Dspring.config.additional-location=./${VOLUME:symphony}/",  "-jar", "wdk.jar", "--spring.profiles.active=${PROFILE:default}"]
```

Comment:
with WORKDIR, can simplify to just java -jar wdk.jar --spring.profiles.active=${PROFILE:default}